### PR TITLE
Allow arbitrary config mixins during test genconf

### PR DESCRIPTION
**Pull request by mellenburg**

### CODE REVIEW

**Comment by malnick** [comment]
@cmaloney This is `Ready for Review`

File: test_util/test_installer_ccm.py [hunk 1]
```diff
@@ -161,6 +165,14 @@ def check_environment():
     options.ci_flags = os.getenv('CI_FLAGS', '')
     options.aws_access_key_id = os.environ.get('AWS_ACCESS_KEY_ID', '')
     options.aws_secret_access_key = os.environ.get('AWS_SECRET_ACCESS_KEY', '')
+
+    add_config_path = os.environ.get('TEST_ADD_CONFIG', '')
```
Comment (cmaloney) [inline test_util/test_installer_ccm.py]:
```python
options.add_config_path = os.getenv('TEST_ADD_CONFIG')
if options.add_config_path:
  assert os.path.isfile(options.add_config_path)
```

**Comment by mellenburg** [comment]
@cmaloney This is ready for review again

**Comment by cmaloney** [comment]
🚢

**Pull request merged**